pkglistgen.py

```diff
@@ -525,7 +525,9 @@ def solve_module(self, groupname, includes, excludes):
             g.ignore(self.groups[e])
 
     def expand_repos(self, project, repo):
-        return StagingAPI(self.apiurl, project).expanded_repos('standard')
+        if not repo:
+            repo = 'standard'
+        return StagingAPI(self.apiurl, project).expanded_repos(repo)
```

Comment:
This would seem like another usecase for `osclib.core` (ie functions that would otherwise fit in `osc.core` and not not specific to staging). They are expected to follow the same format (ie. first parameter of `apiurl`): `expanded_repos(apiurl, project, repo)`.

So the `repo` argument was never used before?!?!? Nice. Perhaps (..., repo = 'standard') as that is a) self-documenting, and b) standard. Unless we explicitely need to support passing `None` which I do not see an example of currently.

Reply:
Yeah, I understand the code is pre-existing, but definitely a case for osclib.core since StagingAPI is being initialized solely to pass `project`.